agent/app/agent_capability.go

```diff
@@ -381,7 +381,7 @@ func (agent *ecsAgent) appendTaskIamRoleCapabilities(capabilities []*ecs.Attribu
 
 func (agent *ecsAgent) appendTaskCPUMemLimitCapabilities(capabilities []*ecs.Attribute, supportedVersions map[dockerclient.DockerVersion]bool) ([]*ecs.Attribute, error) {
 	if agent.cfg.TaskCPUMemLimit.Enabled() {
-		if _, ok := supportedVersions[dockerclient.Version_1_22]; ok {
```

Comment:
a couple of questions:
- shouldn't we keep the granular versioning for these capability advertisements?  What if we need to differentiate between API versions in future?
- customers who update the agent in place on an already-registered container instance would see these capabilities still advertised -- could this lead to any issues?

Reply:
agent in-place update to this version will essentially result in the agent going from supporting docker 1.9.0+ (api version 1.17) to supporting docker 1.12.0+ (api version 1.24). We will need to update documentation but this shouldn't cause significant issues, as this version of docker is very old and not even available on Amazon Linux nor any recent versions of commons OSes (such as Ubuntu)